frontend/src/components/OrderTable/OrderTable.js

```diff
@@ -1,39 +1,61 @@
-import React from 'react';
+import React, { Fragment, useState } from 'react';
+import { useLocation } from 'react-router-dom';
 import './OrderTable.scss';
 import { useTranslation } from 'react-i18next';
+import Button from 'components/Button/Button';
 
 function OrderTable(props) {
+  const [ready, OnReady] = useState(false)
+  const { pathname: pathName} = useLocation();
+  const matchPath = pathName === '/orders/overview';
```

Comment:
there is https://codedaily.io/tutorials/48/Use-matchPath-to-Match-Nested-Route-Paths-in-Parent-Routes-with-React-Router match path for this, especially when you add the ID in the url